src/gui/previewselectdialog.cpp

```diff
@@ -48,6 +48,9 @@
 
 #define SETTINGS_KEY(name) u"PreviewSelectDialog/" name
 
+#define SORT_ROLE Qt::UserRole+1
```

Comment:
We use `enum` for this purpose, e.g.:
https://github.com/qbittorrent/qBittorrent/blob/87644441ad88b251913402ece9fd4b6b78f59ff0/src/gui/trackerlist/trackerlistmodel.h#L81-L84

Reply:
You can just put suggested code in this place.